client/reader/search-stream/suggestion-provider.jsx

```diff
@@ -11,10 +11,11 @@ import { map, sampleSize } from 'lodash';
 import { getLocaleSlug } from 'lib/i18n-utils';
 import { suggestions } from 'reader/search-stream/suggestions';
 import getReaderFollowedTags from 'state/selectors/get-reader-followed-tags';
-import analytics from 'lib/analytics';
+import createRandomId from '@automattic/calypso-analytics';
```

Comment:
I don't see where this function is exported as a default ...

Reply:
Probably should be exported from https://github.com/Automattic/wp-calypso/pull/39001/files#diff-653824456d7b9c1440ed064c9ad1abecR65, then imported like so: ?

`import { createRandomId } from '@automattic/calypso-analytics';
`